app/screens/contacts-screen/contacts.tsx

```diff
@@ -99,12 +98,12 @@ export const ContactsScreen: ScreenType = ({ navigation }: Props) => {
   const { hasToken } = useToken()
   const [matchingContacts, setMatchingContacts] = useState([])
   const [searchText, setSearchText] = useState("")
-  const [isRefreshed, setIsRefreshed] = useState(false)
 
-  const { loading, data, error, refetch } = useQuery(
+  const { loading, data, error } = useQuery(
     gql`
       query contacts {
         me {
+          id
```

Comment:
This is what fixed the balance 0 problem. Was not easy to find.

Reply:
Is this because the cache is stored using this `id` as the key?  So if you don't provide one then it will become `null` on every refresh?